pcstac/Dockerfile

```diff
@@ -22,4 +22,8 @@ RUN --mount=type=cache,target=/root/.cache \
 ENV APP_HOST=0.0.0.0
 ENV APP_PORT=81
 
-CMD uvicorn pcstac.main:app --host ${APP_HOST} --port ${APP_PORT} --log-level info
+# This value should match that which is used as the root_path in FastAPI, which
+# is typically set via the APP_ROOT_PATH environment variable.
+ENV APP_ROOT_PATH=""
+
+CMD uvicorn pcstac.main:app --host ${APP_HOST} --port ${APP_PORT} --root-path ${APP_ROOT_PATH} --log-level info
```

Comment:
Similarly, this fixes the issue in the deployed environment. `APP_ROOT_PATH` is already set in the Helm chart.

Reply:
Just confirming the behavior here: The "default" `APP_ROOT_PATH` is set to `""`, but if the Helm chart defines it it'll override that?

And is the default of `""` is correct? Not `"/"`?

Reply:
In practice, I didn't want to give it a default value. Locally and on PCT, it's `/stac` and in PC its `/api/stac/v1`.  The value is always set by either docker-compose or helm. I believe you need to declare the ENV in order for it be available when the container is run.